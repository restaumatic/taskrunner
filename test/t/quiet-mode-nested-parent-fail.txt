# quiet
echo "Toplevel output before nested call"
taskrunner -n nested echo "Nested task succeeds"
echo "Toplevel output after nested call"
echo "This is the last line before failure"
exit 1