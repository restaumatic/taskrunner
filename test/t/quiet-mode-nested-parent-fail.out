-- output:
[toplevel] stdout | Toplevel output before nested call
[toplevel] stdout | Toplevel output after nested call
[toplevel] stdout | This is the last line before failure
-- exit code: 1
